integration_tests/src/main/python/qa_nightly_sql.py

```diff
@@ -194,7 +194,7 @@
 ("SELECT * FROM test_table WHERE strF LIKE 'Y%'", "* WHERE strF LIKE 'Y%'"),
 ("SELECT * FROM test_table WHERE strF LIKE '%an' ", "* WHERE strF LIKE '%an'"),
 ("SELECT REPLACE(strF, 'Yuan', 'Eric') FROM test_table", "REPLACE(strF, 'Yuan', 'Eric')"),
-("SELECT REGEXP_REPLACE(strF, 'Yu', 'Eric') FROM test_table", "REGEXP_REPLACE(strF, 'Yu', 'Eric')"),
+# ("SELECT REGEXP_REPLACE(strF, 'Yu', 'Eric') FROM test_table", "REGEXP_REPLACE(strF, 'Yu', 'Eric')"),
```

Comment:
Not that we do this elsewhere in the file but it would be great to add comments as to why something is commented out. Or even better just delete it and explain in the PR description and commit why

Reply:
I'm considering what might be the better solution, but wanted to get the PR going. Another option might be to separate out into separate test function in `qa_nightly_select_test.py`. Wonder if that would make more sense.